cargo-near/src/util/mod.rs

```diff
@@ -73,25 +74,52 @@ where
 
     cmd.arg(command);
     cmd.args(args);
+    // Ensure that cargo uses color output for piped stderr.
+    cmd.args(["--color", "always"]);
```

Comment:
Ideally, we should expose this through our CLI too. Doesn't have to happen in this PR though.

`--color auto,always,never`

Reply:
https://github.com/near/cargo-near/issues/60

Reply:
Fixed in #86.